src/BenchmarkDotNet/Locators/ILocator.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.IO;
+
+namespace BenchmarkDotNet.Locators;
+
+public interface ILocator
```

Comment:
We're always returning a `FileInfo`, so a better interface name is in order.

```suggestion
public interface IFileLocator
```